src/components/Progress/HorizontalProgressBar.tsx

```diff
@@ -65,7 +65,7 @@ export default function ({ forgroundColor, backgroundColor, progress, selectedIn
             styles.shadow,
             styles.rightBorder,
             progress < 0.5 && styles.leftBorder,
-            selectedIndex === 1 && { shadowColor: colors.primaryAccent100 },
+            selectedIndex === 1 && { shadowColor: colors.dogerBlue },
```

Comment:
Is this supposed to be "dodgerBlue"? It's not a show stopper, just maybe a spelling mistake.

Reply:
fixed spelling mistake.